platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -111,6 +118,57 @@ public static Result transform(Node nodeXml, TransformerProperties transformerPr
         return transform(new DOMSource(nodeXml), transformerProperties, result);
     }
 
+    /**
+     * @param xml The XML whose root namespace you want
+     * @return Root Namespace
+     */
+    public static String getRootNamespace(String xml) {
+
+        String rootNamespace = null;
+
+        if (xml == null) {
+            return null;
+        }
+
+        initializeXMLInputFactory();
+
+        XMLStreamReader xmlStreamReader = null;
+
+        try (StringReader strReader = new StringReader(xml)) {
+            xmlStreamReader = xmlInputFactory.createXMLStreamReader(strReader);
+
+            while (xmlStreamReader.hasNext()) {
+                int event = xmlStreamReader.next();
+                if (event == XMLStreamConstants.START_ELEMENT) {
+                    rootNamespace = xmlStreamReader.getNamespaceURI();
+                    xmlStreamReader.close();
```

Comment:
It is redundant to close the stream here and also in the finally. You should be able to remove this.